VTEX - Marketplace Protocol.json

```diff
@@ -2781,6 +3094,1533 @@
                         }
                     }
                 }
+            },
+            "EnqueueNewOrderRequest": {
+                "title": "EnqueueNewOrderRequest",
+                "required": [
+                    "marketplaceOrderId",
+                    "marketplaceOrderStatus",
+                    "marketplacePaymentValue",
+                    "connectorEndpoint",
+                    "allowFranchises",
+                    "pickupAccountName",
```

Comment:
`connectorEndpoint` is not a required field if the partner uses our App Template solution, since they already define this field there. Since it's only required if they don't use App Template, we defined as optional in the API.

Same thing with `pickupAccountName`, it's only required for pickup-in-point orders from franchise accounts.

Reply:
Agreed! Included in one of the commits